django_ibmi/creation.py

```diff
@@ -16,19 +16,16 @@
 # | Authors: Ambrish Bhargava, Tarun Pasrija, Rahul Priyadarshi              |
 # +--------------------------------------------------------------------------+
 
-import sys
 try:
     from django.db.backends.creation import BaseDatabaseCreation
 except ImportError:
     from django.db.backends.base.creation import BaseDatabaseCreation
-from django.core.management import call_command
-from django.db.backends.utils import truncate_name
 
-dbms_name = 'dbms_name'
-TEST_DBNAME_PREFIX = 'test_'
 
+class DatabaseCreation(BaseDatabaseCreation):
+    def _clone_test_db(self, suffix, verbosity, keepdb=False):
+        pass
```

Comment:
Seems like if we don't implement this, it should throw an exception, not do nothing.